sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/ComplexQueriesTest.java

```diff
@@ -121,311 +114,6 @@ public void smallTaxiTest() {
         this.addRustTestCase("ComplexQueriesTest.smallTaxiTest", compiler, getCircuit(compiler));
     }
 
-    // Also compiles the code using the Rust handle API
-    @Test
```

Comment:
this test was almost identical with the next one, removed the simpler copy and moved the other one to a separate file